validator/client/metrics.go

```diff
@@ -201,6 +201,17 @@ var (
 			"pubkey",
 		},
 	)
+	// ValidatorInNextSyncCommitteeGaugeVec used to track whether validator is in the next sync committee.
+	ValidatorInNextSyncCommitteeGaugeVec = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "validator",
+			Name:      "in_next_sync_committee",
+			Help:      "validator next sync committee.New in Altair hardfork",
```

Comment:
Super tiny nitpick, same for line 198:
```suggestion
			Help:      "validator next sync committee. New in Altair hardfork",
```